Lab-6.ipynb

```diff
@@ -0,0 +1,631 @@
+      "text/plain": [
+       "<Figure size 1296x576 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# now plot the data resulting from the signal injection\n",
+    "fig, (ax1,ax2) = plt.subplots(1,2, figsize=(18,8),gridspec_kw={'wspace': 0.4})\n",
+    "plt.style.use('ggplot')\n",
+    "\n",
+    "ax1.set_title(\"Signal Injected Data, Background $\\mu=100$\",fontsize=20)\n",
+    "ax2.set_title(\"Log Signal Injected Data, Background $\\mu=100$\",fontsize=20)\n",
+    "ax1.set_xlabel('Observed Events',fontsize=20), ax2.set_xlabel('Observed Events',fontsize=20)\n",
+    "ax1.set_ylabel('Counts',fontsize=20), ax2.set_ylabel('Log(Probability)',fontsize=20)\n",
+    "ax1.tick_params(labelsize=14), ax2.tick_params(labelsize=14)\n",
+    "\n",
+    "ax1.hist(inject,color='b',bins=400,label='Signal injection')\n",
+    "ax2.hist(inject,color='b',density=True,bins=100,label='Signal injection')\n",
+    "ax2.set_yscale('log')\n",
+    "\n",
+    "ax1.legend(loc='best',fontsize=16)\n",
+    "ax2.legend(loc='upper right',fontsize=16)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These histograms show us the background and the observed data distributions post-signal injection. The log plot shows more clearly the true distribution shapes and the minor changes along the curves; as we can see, both the background and the observation shapes appear parabolic on the log plot and therefore approached a Gaussian distribution.\n",
+    "\n",
+    "After signal injection into the background, there are two notable changes to the histogram distributions: One, the peaks have changed. Both curves have shorter peaks (in counts) than pre-signal injection. Second, most importantly, the observed signal has changed. While the background remains the same as expected, since our fake background doesn't change with time or any other expected changes, the signal histogram shape has now shifted up on the x-axis. Looking closer, we see the peak, where the mean is located, has shifted up exactly by 100, from 215 to 315.\n",
```

Comment:
Yes there's a shift with an additive effect after the injection. I really enjoy the fourth paragraph, really well worded on what's actually happening.